modules/admin-credentials-root-secret-formats.adoc

```diff
@@ -41,13 +32,11 @@ metadata:
   namespace: kube-system
   name: aws-creds
 stringData:
-  aws_access_key_id: <AccessKeyID>
-  aws_secret_access_key: <SecretAccessKey>
+  aws_access_key_id: <access_key_id>
+  aws_secret_access_key: <secret_access_key>
```

Comment:
Considering the RHOSP secret format i.e. clouds.yaml: <base64-encoded_cloud_creds> and to make all of these secret formats consistent, maybe <base64-access_key_id> is better?
This comment is applicable for secrets of AWS/Azure/GCP.

Reply:
I left it the same as in the dev readme - @akhil-rane is there any reason to not accept Jianping's suggestion that I am just unaware of? If not, I will update all of these as suggested :slightly_smiling_face: